### PR TITLE
Add an IMU tare service

### PR DESCRIPTION
**Pull request by nakai-omer**

### CODE REVIEW

File: vectornav/src/vectornav.cc [hunk 1]
```diff
@@ -487,6 +494,27 @@ class Vectornav : public rclcpp::Node
...
+    if (!vs_) {
+      RCLCPP_ERROR(get_logger(), "Tare requested before sensor has been initialized");
+    }
+
+    try {
+      vs_->tare();
+    } catch (const std::exception & e) {
+      RCLCPP_ERROR_STREAM(get_logger(), "Failed to tare. error: " << e.what());
+    } catch (...) {
+      RCLCPP_ERROR(get_logger(), "Failed to disable async output. error: unknown");
```
Comment (dawonn) [inline vectornav/src/vectornav.cc]: What does this have to do with async output?

**Comment by dawonn** [comment]
Thanks for the PR!

Can you please add a comment to the PR describing how you tested the change?

File: vectornav/src/vectornav.cc (hunk 1 above)
Comment (nakai-omer) [reply vectornav/src/vectornav.cc]: oops stayed there by mistake, fixed

**Comment by nakai-omer** [comment]
@dawonn Sorry for late response. To test, we have visualized the imu data in rviz, and sent a tare request from the ros2 service cli command, and we saw how the imu resets to 0,0 position and orientation.

**Review by dawonn — APPROVED** [review]
Thanks!

**Comment by dawonn** [comment]
Will merge after the other PR testing is completed.

**Comment by dawonn** [comment]
Any updates on the testing?

**Comment by dawonn** [comment]
Please rebase, the diff seems to show unrelated content ENU*

**Comment by dawonn** [comment]
Closing after a year of silence.
